packages/expo-cli/src/commands/utils/validateDependenciesVersions.ts

```diff
@@ -71,10 +75,39 @@ export async function validateDependenciesVersionsAsync(
           )}`
       );
     }
+  }
 
-    return false;
+  if (Versions.gteSdkVersion(exp, '45.0.0')) {
+    try {
+      /* This will throw if the dependency looked for is not installed,
+       * but that doesn't apply here, so that error is ignored */
+      const packageVersionConfigPlugins = await resolvePackageVersionsAsync(projectRoot, [
```

Comment:
this will only find a single version of the dep, right? if you have libraries in your project that each depend on a different version of `@expo/config-plugins` that may not be surfaced here

Reply:
Yes, it will loop through the flat list of packages in node_modules, so if will only find one. Still, if there _is_ one with the wrong version there, and it warns about it, then the next step is to use `yarn why` or `npm why` which will turn up everything.